package.json

```diff
@@ -28,7 +28,11 @@
     "security-checks": "cross-env NODE_PATH=. grunt security-checks",
     "healthcheck": "cross-env NODE_PATH=. mocha test/healthCheck.js --timeout 15000 --reporter spec",
     "post-data": "cross-env NODE_PATH=. node post-data.js",
-    "sonar-scan": "sonar-scanner"
+    "sonar-scan": "sonar-scanner",
+    "test:coverage": "yarn precoverage && yarn coverage && yarn postcoverage",
```

Comment:
You shouldn't explicitly call `yarn precoverage ` or `yarn postcoverage ` as they are both called internally when running `yarn coverage`.

Reply:
Generally speaking we don't want a `package.json` that contains scripts that fulfils both strategic (i.e. CNP) and tactical (i.e. Bart and Jenkins pipeline). We should go with CNP and remove all the tactical scripts. In short, we don't want both in the same file.